src/utils/validators.test.js

```diff
@@ -0,0 +1,69 @@
+// ------------------------------------------------------------------------------ //
```

Comment:
This header is not necessary :D, we use `.test` suffix to inform that this is a file with tests